apps/i18n/common/fr_fr.json

```diff
@@ -750,9 +753,18 @@
   "loginInfo_noStudents": "* Il semble que vous n'ayez aucun étudiant dans cette section! Ajoutez des étudiants dans l'onglet [Gérer les étudiants] ({url}) de cette section. *",
   "loginInfo_oauthSectionCodes": "Les sections {fournisseur} n'ont pas de code de section à 6 chiffres, votre élève n'aura donc pas besoin d'utiliser un code de section pour se connecter.",
   "loginInfo_resetTitle": "Réinitialiser le mot de passe des étudients",
+  "loginInfo_resetPasswordBody": "Si votre élève a créé un compte Code.org en utilisant son adresse électronique et son mot de passe (plutôt que de se connecter via Google, Facebook, Microsoft ou Clever), vous pouvez réinitialiser son mot de passe en accédant à l'onglet [Manage Students] ({url}). de votre section et en cliquant sur le bouton \"Réinitialiser le mot de passe\" dans la colonne \"Mot de passe\".",
+  "loginInfo_resetSecretBody": "Vous pouvez réinitialiser le secret de n'importe lequel de vos étudiants en accédant à l'onglet [Gérer les étudiants] ({url}) de la section, en cliquant sur le bouton \"Afficher le secret\" sous la colonne \"Mot de passe\", puis en cliquant sur le bouton \"Réinitialiser le secret\". .",
```

Comment:
not a blocker for this sync, but looks like we got some broken markdown here

Reply:
Good catch. Fixed in Crowdin.